salt/_modules/caasp_filters.py

```diff
@@ -40,3 +40,15 @@ def get_max(seq):
     # TODO: use the builtin filter (https://docs.saltstack.com/en/latest/topics/jinja/index.html#max)
     #       once we Salt>2017.7.0
     return max(seq)
+
+
+def is_admin_node():
+    '''
+    Returns true if the node has the 'admin' and/or the 'ca'
+    roles.
+
+    Returns false otherwise
+    '''
+
+    node_roles = __salt__['grains.get']('roles', [])
```

Comment:
It would be safer to use [this function](https://github.com/kubic-project/salt/blob/master/salt/_modules/caasp_grains.py#L12-L20) (so `node_roles = __salt__['caasp_grains.get']('roles', [])`. Otherwise this will crash is someone tries to use `is_admin_node()` in a `orch/*.sls` file.

Reply:
As discussed we are not going to do that, this is not needed because right now this function is only called on the nodes that are executing the states.